dbms/CMakeLists.txt

```diff
@@ -179,6 +179,7 @@ target_link_libraries (tiflash_common_io
     ${LINK_LIBRARIES_ONLY_ON_X86_64}
     ${LZ4_LIBRARY}
     ${ZSTD_LIBRARY}
+    ${QPL_LIBRARY}
```

Comment:
this line should be put to
```
if (USE_QPL)
    target_link_libraries(tiflash_common_io ${QPL_LIBRARY})
... 
```

Reply:
When USE_QPL = 0, ${QPL_LIBRARY} is empty, I think there is no need to add if branch.

Reply:
I think put it to the if branch is more clear

Reply:
Remove this line, and 

```
if (USE_QPL)
    target_link_libraries (tiflash_common_io PUBLIC ${QPL_LIBRARY}) <<--- add here
    target_include_directories (dbms BEFORE PRIVATE ${QPL_INCLUDE_DIR})
    set_source_files_properties (
        src/IO/Compression/CompressionCodecDeflateQpl.cpp
        PROPERTIES COMPILE_FLAGS "-mwaitpkg")
endif ()
```